include/hipSYCL/sycl/queue.hpp

```diff
@@ -606,6 +606,26 @@ class queue : public detail::property_carrying_object
       cgh.memcpy(dest, src, num_bytes);
     });
   }
+  
+  template <typename T>
+  event copy(const T* src, T* dest, std::size_t count) {
+    return this->memcpy(reinterpret_cast<void*>(dest), 
+      reinterpret_cast<const void*>(src), count * sizeof(T));
```

Comment:
I think a `static_cast` should be sufficient to convert to `void*`. No need to bring out the big guns :)

Reply:
Fixed.